test/basic-querying.test.js

```diff
@@ -234,21 +258,24 @@ describe('basic-querying', function() {
       User.find({order: 'name DESC'}, function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users.pop().name.should.equal('George Harrison');
-        users.pop().name.should.equal('John Lennon');
-        users.shift().name.should.equal('Stuart Sutcliffe');
+        bdd.itIf(!isCassandra, 'in-place sort is supported', function() {
+          users.pop().name.should.equal('George Harrison');
+          users.pop().name.should.equal('John Lennon');
+          users.shift().name.should.equal('Stuart Sutcliffe');
+        });
         done();
       });
     });
 
     it('should query sorted desc by order integer field even though there' +
-        'is an async model loaded hook', function(done) {
+      'is an async model loaded hook', function(done) {
```

Comment:
remove one more indent if you want to follow the style guide's style: http://loopback.io//doc/en/contrib/style-guide.html#callback-function-moved-to-the-next-line